index.html

```diff
@@ -22,36 +22,47 @@
           id="headeline-bg"
         />
         <header id="page-head">
-          <div id="header-logo">
-            <a href="#" id="logo">Welcome</a>
-          </div>
           <nav id="navbar">
-            <div id="nav-links">
-              <a href="#" class="nav-link"
-                ><i class="fa fa-bars" aria-hidden="true"></i
-              ></a>
-              <ul id="menu-items">
-                <li>
-                  <a href="#" class="nav-link">Portfolio</a>
-                </li>
-                <li>
-                  <a href="#" class="nav-link">About</a>
-                </li>
-                <li>
-                  <a href="#" class="nav-link">Contact</a>
-                </li>
-              </ul>
+            <div id="nav-top">
+              <div id="header-logo">
+                <a href="#" id="logo">Welcome</a>
+              </div>
+              <div id="nav-links">
+                <a href="#" class="nav-link"
+                  ><i class="fa fa-bars" aria-hidden="true"></i
+                ></a>
+              </div>
             </div>
+            <div class="menu">
```

Comment:
- [ ] Please make sure when the user clicks any option of the mobile menu(Portfolio, About, Contact), the mobile menu disappears.